src/main/kotlin/com/chattriggers/ctjs/minecraft/objects/display/DisplayHandler.kt

```diff
@@ -20,12 +21,31 @@ object DisplayHandler {
     fun clearDisplays() = displays.clear()
 
     @SubscribeEvent
-    fun renderDisplays(event: RenderGameOverlayEvent.Text) {
+    fun renderDisplayOverlay(event: RenderGameOverlayEvent.Text) {
         GlStateManager.pushMatrix()
-        displays.forEach(Display::render)
+        displays.forEach {
+            if(it.registerType == RegisterType.RENDEROVERLAY) {
+                it.render()
+            }
+        }
         GlStateManager.popMatrix()
     }
 
+    @SubscribeEvent
+    fun renderDisplayGui(event: GuiScreenEvent.DrawScreenEvent.Post) {
+        GlStateManager.pushMatrix()
+        displays.forEach {
+            if(it.registerType == RegisterType.POSTGUIRENDER) {
+                it.render()
+            }
+        }
+        GlStateManager.popMatrix()
+    }
+
+    enum class RegisterType {
+        RENDEROVERLAY, POSTGUIRENDER, OTHER
```

Comment:
`OTHER` doesn't seem necessary, Displays aren't necessary supposed to be insanely customizable, so let's just add the post gui render thing here.

Reply:
Also I think it would be better to use SCREAMING_SNAKE_CASE to be consistent with DisplayHandler.Background.

Reply:
OTHER has been removed and the Enum classes have been made consistent with DisplayHandler's other Enum classes. The files have been changed to accommodate this change and it's been rebased to a single commit